src/components/useraccount/SidebarMenu.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <div class="my-account-sidebar">
     <div id="my-account-mobile-content"
-          class="my-account-sidebar-items active">
-      <span>Personal Details</span>
+          class="my-account-sidebar-items"
+          :class="{ active: showTab.showPersonalDetails }">
+          <router-link to="/user">
```

Comment:
Indentation

Reply:
Also, never use router-link going to a URL. Only exception is "/", because is universal. If someone wants to change the URL to go to "my account", they should only have to change it in routes file, not in every single component that has a link to this page. Therefore, the best practice is to use "name".

Reply:
https://router.vuejs.org/guide/essentials/named-routes.html